src/wp-includes/class-wp-customize-manager.php

```diff
@@ -2501,7 +2501,11 @@ public function save() {
 				}
 				$changeset_date_gmt = get_gmt_from_date( $changeset_date );
 			} else {
-				$timestamp = strtotime( $changeset_date );
+				try {
+					$timestamp = ( new DateTimeImmutable( $changeset_date ) )->getTimestamp();
+				} catch ( \Exception $exception ) {
+					$timestamp = false;
```

Comment:
Seeing you've added an exception handler here. Shouldn't this happen throughout the hole code? Or maybe this could be solved by a wrapper function like

```
function wp_get_timestamp( $string ) {
  try {
    $timestamp = ( new DateTimeImmutable( $string ) )->getTimestamp();
  } catch ( Exception $e ) {
    // Some exception logging here.
    $timestamp = strtotime( $string );
  }
  return $timestamp;
}
```